mopidy_musicbox_webclient/static/js/controls.js

```diff
@@ -424,15 +424,19 @@ function triggerVolume() {
     volumeChanging = 0;
 }
 
-function toggleMute() {
-    mopidy.mixer.getMute().then(function(mute) {
-        mopidy.mixer.setMute(!mute);
+function setMute(nwmute) {
```

Comment:
Nitpicking really but it would be clearer to call this something like `updateMuteIcon` and distinguish it from `doMute`.  I think there are some things that follow a convention like this, but I might be remembering that wrong. It would be nice to have one.